src/style.scss

```
@@ -0,0 +1,126 @@
+$main-color: #060b35 ;
+$secondary-color: #616070 ;
+
+* {
+  margin: 0;
```

```suggestion
   box-sizing: border-box;
```
it is just one property which generally use in * selector
reset default margins only for elements whert it is necessary